docs/src/Components.jsx

```diff
@@ -19,7 +20,15 @@ var Components = React.createClass({
           <h4>Block</h4>
           <p>Create block level buttons by adding <code>size='block'</code>.</p>
           <CustomComponent codeText={fs.readFileSync(__dirname + '/../examples/ButtonBlock.jsx', 'utf8')} />
-
+          <h4>Attributes</h4>
+          <ul>
+            <li><code>purpose</code> String - The purpose of the button</li>
```

Comment:
should we document expected values for these fields? It would save people digging through the code or looking at warnings to see what values are allowed in the propTypes